### PR TITLE
Copy to clipboard

### PR DESCRIPTION
**Pull request by noodlesftw**
Added a copy-to-clipboard feature when the user hits the `enter` key. It uses the `copy-paste` package which covers macOS, Linux and Windows.

### CODE REVIEW

**Comment by radiovisual** [comment]
So this PR simply copies all 7 emojis to the clipboard? Issue #2 has an open discussion going on the best way to get emojis into the clipboard, with a focus on how to copy a single emoji, since a single emoji is likely to be the majority use case.

**Comment by noodlesftw** [comment]
@radiovisual yes, it copies all 7 to the clipboard. I saw the discussion after I created the PR. I'm planning on building that feature later today.

File: cli.js [hunk 1]
```diff
@@ -67,7 +72,23 @@ process.stdin.on('keypress', (ch, key) => {
 		query.pop();
 	} else if (key.ctrl && key.name === 'u') {
 		query.length = 0;
-	} else if (key.name === 'return' && prevResult) {
+	} else if (key.ctrl && key.name === '0') {
+		npc.copy(prevResult);
+	} else if (key.ctrl && key.name === '1') {
```
Comment (sindresorhus) [inline cli.js]: Too much repetition. Just do `key.ctrl && Number(key.name) >= 1 && Number(key.name) <= 7` and `prevResult[Number(key.name) - 1]`.

Comment (sindresorhus) [inline cli.js]: You can make `Number(key.name)` a variable so it can be reused.

File: cli.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ const chalk = require('chalk');
 const debounce = require('lodash.debounce');
 const hasAnsi = require('has-ansi');
 const mem = require('mem');
+const npc = require('copy-paste');
```
Comment (sindresorhus) [inline cli.js]: npc => copyPaste

File: cli.js [hunk 3]
```diff
@@ -42,7 +43,12 @@ dns.lookup('emoji.getdango.com', err => {
 		logUpdate(`\n${chalk.bold.red('› ')}${chalk.dim('Please check your internet connection')}\n\n`);
 		process.exit(1);
 	} else {
-		logUpdate(`${pre}${chalk.dim('Relevant emojis will appear when you start writing')}\n\n`);
+		let introText = '';
+		introText += `${pre}Relevant emojis will appear when you start writing.`;
+		introText += `${pre}You can copy an emoji by holding down the CTRL key and pressing a number from 1 - 7.`;
```
Comment (sindresorhus) [inline cli.js]:
```
introText += `${pre}You can copy an emoji by holding down the Ctrl key and pressing a number from 1 to 7, or 0 to copy all.`;
```

File: cli.js (hunk 1 above)
Comment (noodlesftw) [inline cli.js]: @sindresorhus yeah, I'm still working on it, was doing some testing yesterday. I don't think the ctrl + number combination is gonna work (on macOS at least). The combination is already assigned or something.

**Comment by sindresorhus** [comment]
@noodlesftw Any chance you could finish this up? :)

**Comment by noodlesftw** [comment]
@sindresorhus yes, will be working on it today. Will have to find another key combination since `ctrl` is assigned on macOS.

**Comment by sindresorhus** [comment]
No problem, we can use Shift or Alt instead.

**Comment by sindresorhus** [comment]
🌴

**Comment by radiovisual** [comment]
Any progress on this PR? My 💻  (terminal) is 😢  (sad), and the only way to cheer it up 🎉  will be to let it auto-copy 😄  (smiley faces) and 🍔  (hamburgers) to the clipboard.

cc: @noodlesftw 🍜

**Comment by sindresorhus** [comment]
Closing so other people can give it a shot. Happy to reopen if you ever get to this @noodlesftw ;)

**Pull request closed without merging**